src/lib/fm-fontutils.c: In function ‘FcListUserDirs’:
src/lib/fm-fontutils.c:111:49: error: ‘W_OK’ undeclared
src/lib/fm-fontutils.c: In function ‘FT_Get_Font_Info’:
src/lib/fm-fontutils.c:205:44: error: ‘W_OK’ undeclared

--- origsrc/font-manager-0.5.7/src/lib/fm-fontutils.c	2010-12-23 21:48:30.000000000 -0600
+++ src/font-manager-0.5.7/src/lib/fm-fontutils.c	2013-01-17 23:33:44.693187500 -0600
@@ -21,6 +21,7 @@
  *   Boston, MA 02110-1301, USA
 */
 
+#include <unistd.h>
 #include <glib.h>
 #include <glib/gprintf.h>
 #include <glib/gstdio.h>
